administrator/components/com_config/src/View/Component/HtmlView.php

```diff
@@ -82,9 +82,18 @@ public function display($tpl = null)
         $this->fieldsets   = $form ? $form->getFieldsets() : null;
         $this->formControl = $form ? $form->getFormControl() : null;
 
-        // Don't show permissions fieldset if not authorised.
-        if (!$user->authorise('core.admin', $component->option) && isset($this->fieldsets['permissions'])) {
-            unset($this->fieldsets['permissions']);
+        // Remove unauthorised preference tabs.
+        foreach ($this->fieldsets as $key => $value) {
+            if ($key == 'permissions') {
+                if ((!$user->authorise('core.admin', $component->option) || !$user->authorise('core.options.permission', $component->option))
+                    && isset($this->fieldsets['permissions'])) {
```

Comment:
```suggestion
                if (
                    (!$user->authorise('core.admin', $component->option) || !$user->authorise('core.options.permission', $component->option))
                    && isset($this->fieldsets['permissions'])
                ) {
```
Now as the first code style checker was happy, the 2nd one (we use 2) found this. I think if this is fixed, code style should be ok.